src/SW_Files.c

```diff
@@ -63,15 +63,14 @@ void SW_CSV_F_INIT(const char *s, LOG_INFO *LogInfo) {
     char inbuf[MAX_FILENAMESIZE];
     char dirString[FILENAME_MAX];
     char *resPtr = NULL;
-    char *endInbuf = inbuf + sizeof inbuf - 1;
 
     DirName(s, dirString);
 
     if (DirExists(dirString)) {
         resPtr =
             (char *) sw_memccpy((void *) inbuf, (char *) s, '\0', sizeof inbuf);

```

Comment:
SW_CSV_F_INIT() calls sw_memccpy() and then fails via reportFullBuffer() if `isnull(resPtr)`; however, it never sets the last byte of inbuf to '\0'. Should we use here instead sw_memccpy_inc() for that?